### ISSUE TITLE
No ability to edit Inline Image alternative text in context

### ISSUE THREAD

**Issue by mrwweb** [WordPress/gutenberg#63323]
### Description

Currently, to the best of my knowledge, it is only possible edit an inline image's global alt text (without editing HTML). This is an issue as most alt text should be contextual. I'm currently in a place where I need my inline images to have empty alt text while it's very important that they retain alt text for use in galleries throughout the site.

### Step-by-step reproduction instructions

1. Insert inline image
2. You can't edit the alt text

### Environment info

- WP 6.6-RC2

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

No
